lib/ansible/module_utils/vmware.py

```diff
@@ -293,9 +453,13 @@ def list_snapshots(vm):
     if vm.snapshot is None:
         return result
 
-    result['snapshots'] = list_snapshots_recursively(vm.snapshot.rootSnapshotList)
+    result['snapshots'] = list_snapshots_recursively(
+        vm.snapshot.rootSnapshotList
+    )
```

Comment:
This should not be necessary for PEP8. We have a line-length of 160, not the default 80.